README.md

```diff
@@ -265,6 +265,21 @@ When this VM is first created, Chrome and Firefox will be upgraded to the latest
 Google package repositories. The ``vagrant provision`` command can be used at a later time to trigger the browser
 upgrade and general VM provisioning mechanism.
 
+### Run Tests In a Docker Container
+
+You can also run the tests from a [Docker](https://docs.docker.com/get-docker) container.
+
+Once you have Docker installed, run the following commands to build a Docker image and start a container:
+
+* Build the image: `docker build -t infusion .`
+* Run the container: `docker run --name infusion -p 8000:80 infusion`
+
+Infusion will be available at [http://localhost:8000](http://localhost:8000)
+
+* To stop and remove the container: `docker rm -f infusion`
+
+If you make changes to Infusion, repeat the steps to build the image and start a new container.
+
 #### Coverage Reporting
```

Comment:
Coverage reporting was a sub section under the VM instructions. It should either be moved ahead of the docker instructions or made as a level 3 heading so that it sits at the same level as the sections about how to run the tests.

Reply:
Fixed, thanks. I didn't notice the extra `#` in the heading, sorry.